python/py-authres/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+set real_name       authres
+name                py-${real_name}
+version             1.2.0
+revision            0
+worksrcdir          ${real_name}-${version}
+distfiles           ${real_name}-${version}${extract.suffix}
+categories-append   email
+license             Apache-2
+maintainers         @BjarneDMat openmaintainer
+description         Authentication-Results Headers generation and parsing for Python/Python3.
+long_description    ${description} \
+    This module (authres) is designed to provide support for RFC 5451/7001, \
+    originally https://tools.ietf.org/html/rfc5451 (and then \
+    https://tools.ietf.org/html/rfc7001 and https://tools.ietf.org/html/rfc7601) \
+    and currently https://tools.ietf.org/html/rfc8601, processing in Python.
+
+platforms           darwin
+homepage            https://launchpad.net/authentication-results-python
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+
+python.versions     27 37
```

Comment:
Is there really a need to create py27 sub-port?

Reply:
depends on whether or not macports wishes to continue it's support for python 2.7. I included it for the sake of full support for both 2.7 & 3.7